examples/api/hello_world_web_server/api.py

```diff
@@ -16,9 +16,11 @@
 """Definition of API endpoints"""
 
 from fastapi import Depends, FastAPI
+from ghga_service_chassis_lib.api import configure_cors
 from .config import get_config
 
 app = FastAPI()
+configure_cors(app, config=get_config())
```

Comment:
Maybe the chassis lib should instead offer a wrapper for constructing the `FastAPI` object based on a config. The current implementation requires that the developer knows that specifying CORS settings in the config additionally requires code changes.

Reply:
Hm, I was trying to avoid that to not loose autocompletion when forwarding arguments. But if you think that's necessary, I will make a draft.

Reply:
Alternatively the function could be renamed to `configure_app` or something along those lines and would always be called.

Reply:
Sure we can also do that. Later we might even add more functionality to it.